cirq/iqm/iqm_device.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""IQM devices https://iqm.fi/devices"""  # TODO: PQC-5
```

Comment:
I'm receiving a `time-out` error on this link.

Reply:
Me too.  I also echo @vtomole concerns regarding the pydocs.  We should improve the documentation before it is ready to merge.

Reply:
Added documentation here: https://github.com/quantumlib/Cirq/pull/2701/commits/c98e112b6ae252d6f167706231e6c37d41dedb2b

Reply:
Please see the current state, is this okay for you?